src/bpf_program.rs

```diff
@@ -70,12 +99,19 @@ impl BpfProgram {
         events_per_second.round() as i64
     }
 
-    pub fn cpu_time_percent(&self) -> f64 {
+    fn cpu_time_percent(&self) -> f64 {
         if self.period_ns == 0 {
             return 0.0;
         }
         self.runtime_delta() as f64 / self.period_ns as f64 * 100.0
     }
+
+    pub fn update_stats(&mut self) {
```

Comment:
Can we sort without having to cache the value? I would prefer not doing this caching unless it was absolutely necessary

Reply:
The sort is now updated without using the caching, and bpf_program.rs should now be reverted to what it previously was as well. 👍